velox/exec/Aggregate.h

```diff
@@ -18,6 +18,7 @@
 #include <folly/Synchronized.h>
 
 #include "velox/common/memory/HashStringAllocator.h"
+#include "velox/core/ExpressionEvaluator.h"
```

Comment:
Could move setLambdaExpressions() to .cpp and forward declare ExpressionEvaluator to break the header/header dependency.